### PR TITLE
Added some tests for GoalModel

### PR DESCRIPTION
**Pull request by iGusev**

### CODE REVIEW

File: tests/Feature/Models/GoalTest.php [hunk 1]
```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace Tests\Feature\Models;
+
+use App\Models\Goal;
+use App\Models\User;
+use Carbon\Carbon;
```
Comment (sbine) [inline tests/Feature/Models/GoalTest.php:7]: `User` and `Carbon` are both unused in this file, could you please delete these imports?

File: tests/Feature/Models/GoalTest.php [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+class GoalTest extends TestCase
+{
+    use RefreshDatabase;
+
+    /** @test */
+    public function goal_get_achieved_attribute_false()
+    {
+        $goal = Goal::factory()->make([
+            'balance' => 0.5,
+            'amount' => 1
```
Comment (sbine) [inline tests/Feature/Models/GoalTest.php:21]: Please add trailing commas to the end of every array item (below too) to match the php-cs-fixer rules

File: tests/Feature/Models/GoalTest.php [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    /** @test */
+    public function goal_get_progress_attribute_300()
+    {
+        $goal = Goal::factory()->make([
+            'balance' => 3,
+            'amount' => 1
+        ]);
+
+        $this->assertIsString($goal->getProgressAttribute());
+        $this->assertEquals(300, $goal->getProgressAttribute());
```
Comment (sbine) [inline tests/Feature/Models/GoalTest.php:60]: I think you found a bug here 😄

**Review by sbine — COMMENTED** [review]
Thanks for your contribution! Just some small linting issues, then this is good to go.

File: tests/Feature/Models/GoalTest.php (hunk 3 above)
Comment (iGusev) [reply tests/Feature/Models/GoalTest.php]: Added some logic for getProgressAttribute() for cases when >100% + fixed test

File: tests/Feature/Models/GoalTest.php (hunk 2 above)
Comment (iGusev) [reply tests/Feature/Models/GoalTest.php]: +

File: tests/Feature/Models/GoalTest.php (hunk 1 above)
Comment (iGusev) [reply tests/Feature/Models/GoalTest.php]: +

**Pull request merged**